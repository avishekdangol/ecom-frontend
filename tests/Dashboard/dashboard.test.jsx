import { render } from '@root/tests/testUtils';
import Dashboard from '@/pages/Dashboard/Index';

describe('Dashboard', () => {
  test('snapshot', () => {
    const { asFragment } = render(<Dashboard />);
    expect(asFragment()).toMatchSnapshot();
  });
});
