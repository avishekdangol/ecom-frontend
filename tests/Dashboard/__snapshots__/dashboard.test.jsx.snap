// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard snapshot 1`] = `
<DocumentFragment>
  <h1>
    Dashboard
  </h1>
</DocumentFragment>
`;
